parachain/pallets/ethereum-beacon-client/tests/fixtures/goerli_block_update.json

```diff
@@ -1,2224 +0,0 @@
-{
```

Comment:
I renamed these test files to `mainnet_` instead of `goerli_`, to rather use the config spec than the actual testnet name.